core/build.gradle

```diff
@@ -71,4 +71,8 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.Kotlin2JsCompile.class).configu
             freeCompilerArgs += "-Xwasm-enable-array-range-checks"
         }
     }
+}
+
+tasks.named("jvmApiBuild") {
```

Comment:
api checks cannot be performed because of kotlin.metadata is not up to date

Reply:
You can also try to pick this commit: https://github.com/Kotlin/kotlinx.serialization/commit/8c1a81e7a701964eb38b30592e4fd339386852af

Reply:
AFAIR, There is the problem with ABI changed when kotlin 1.9 emits new method `entries` for enums. So the test data should be fixed or this feature should be disabled.